source/UIAHandler/__init__.py

```diff
@@ -49,6 +49,14 @@
 from . import remote as UIARemote
 
 
+baseCachePropertyIDs = set([
+	UIA.UIA_FrameworkIdPropertyId, UIA.UIA_AutomationIdPropertyId, UIA.UIA_ClassNamePropertyId,

```

Comment:
Could these be separate to 1 per new line?
It improves visual readability and makes diffs simpler if items are added/removed.